packages/contracts/test/test-helpers/data/index.ts

```diff
@@ -1,5 +1,38 @@
 import { create2Tests } from './create2.test.json'
 import { rlpTests } from './rlp.test.json'
+import * as fs from 'fs'
+
+const createSynthetixJSON = () => {
+  const files = {}
+  const sanitizeLibs = (str: string): string => {
+    return str
+      .split('__$')
+      .join('000')
+      .split('$__')
+      .join('000')
+  }
+  const dir = __dirname + '/synthetix/optimized/'
+  console.log(dir)
+  fs.readdirSync(dir).forEach((fileName) => {
+    if (fileName.endsWith('.json')) {
+      const obj = JSON.parse(fs.readFileSync(dir + fileName, 'utf8'))
+      files[fileName] = {
+        bytecode: '0x' + sanitizeLibs(obj.evm.bytecode.object),
+        deployedBytecode: '0x' + sanitizeLibs(obj.evm.deployedBytecode.object),
+      }
```

Comment:
What is the difference between `bytecode` and `deployedBytecode`?

Reply:
bytecode is the same as init code - it includes the constructor code, which is only run once during the contract's creation. Deployed bytecode is the code that actually ends up stored at the contract address. https://medium.com/authereum/bytecode-and-init-code-and-runtime-code-oh-my-7bcd89065904

Reply:
Thanks for sharing that article!